src/appmixer/ai/claude/package.json

```diff
@@ -1,5 +0,0 @@
-{
-  "dependencies": {
-    "langchain": "0.3.24"
```

Comment:
why you've removed the package.json? langaing is used in the `TransformTextToJSON`